trophies.toml

```diff
@@ -78,6 +78,11 @@ description = 'Solve any hole in both Perl and Raku.'
 
 # Miscellaneous
 
+[[Miscellaneous]]
+name        = 'Different Strokes'
+emoji       = '🌈'
+description = 'Submit unique solutions for bytes and chars on the same hole.'
```

Comment:
Can we reword this to be more like the others, I suggest:

Solve any hole using different bytes and chars solutions.